sass/partials/_time_to_retirement.scss

```diff
@@ -449,5 +519,4 @@ input[type="text"].form-control {
     position: absolute;
     left: 15%;
     right: 15%;
-}
-
+}
```

Comment:
500 lines of css seems like a lot for one view. There might be a way to consolidate some of this into classes that are re-usable throughout this view. Not a huge deal though.